docs/blockchain/contracts/nft-contract/nft-actions/rmfctofr.a.experimental.md

```diff
@@ -0,0 +1,58 @@
+---
+title: 'rmfctofr.a'
+order: 42
+
+---
+
+# rmfctofr.a
+
+Cancel the buy offer made on the token factory.
+
+## Technical Behavior
+
+Note that `rmfctofr.a` is used to cancel the offer made by `mkfctofr.a` only.
+
+A buyer can cancel any of their offers at any time.
+
+Once the offer is expired, anyone can cancel the offer.
+
+When the offer is canceled, the offered price will be transferred from eosio.nftofr account back to buyer.
+
+## Action Parameters
+
+| Property Name | C++ Type | JavaScript Type | Definition                    |
+| ------------- | -------- | --------------- | ----------------------------- |
+| canceler      | name     | String          | Account who cancel an offer   |
```

Comment:
Account who cancel an offer
should be
Account who cancels an offer

Reply:
updated